src/Pages/Infrastructure/Create/index.jsx

```
@@ -53,7 +54,14 @@ const CreateInfrastructureMonitor = () => {
 	const monitorState = useSelector((state) => state.infrastructureMonitor);
 	const dispatch = useDispatch();
 	const navigate = useNavigate();
-	const { t } = useTranslation();
+	const { monitorId } = useParams();
+	const { t } = useTranslation();	
+
+	// Determine if we are creating or editing
+	const isCreate = typeof monitorId === "undefined";
+
+	// Fetch monitor details if editing
+	const { monitor, isLoading, networkError } = isCreate ? {} : useHardwareMonitorsFetch({ monitorId });
```

ESLint

[error] 'isLoading' is assigned a value but never used.

(no-unused-vars)